build/travis/configure-travis-ci-xsite.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+
+. build/common.sh
+
+export MAKE_DATADIR_WRITABLE=true
+export GO111MODULE=on
+export INITCONTAINER_IMAGE=quay.io/quay/busybox:latest
+export TESTING_NAMESPACE=${TESTING_NAMESPACE-namespace-for-testing}
+export KUBECONFIG=~/kind-kube-config.yaml
+KIND_VERSION=${1-${KIND_VERSION}}
+METALLB_VERSION=${2-${METALLB_VERSION}}
+YQ_VERSION=4.6.1
+
+#Ensure that GOPATH/bin is in the PATH
+if ! validateYQ; then
+  printf "Valid yq not found in PATH. Trying adding GOPATH/bin\n"
+  PATH=$(go env GOPATH)/bin:$PATH
+  if ! validateYQ; then
+    installYQ
+  fi
+fi
+
+if ! [ -x "$(command -v kind)" ]; then
+  curl -Lo kind https://github.com/kubernetes-sigs/kind/releases/download/"${KIND_VERSION}"/kind-linux-amd64 && chmod +x kind && sudo mv kind /usr/local/bin/
```

Comment:
I would download kind & kubectl to `/tmp/bin` (or some other temporary directory) and added it to `PATH`.
It is just to avoid using `sudo` :)

Reply:
Stored kubectl and kind commands into the /tmp/k8s/bin/ if not present